snippets/misp-warninglists/misp-warninglists.storm

```diff
@@ -0,0 +1,96 @@
+/*
+Name: misp-warninglists.storm
+Inspired by: cybermail2822@gmail.com
+Modified by: reign@vertex.link
+Date: 2023-10-09
+
+Description: Ingests and tags MISP warninglists data. At the time of release this snippet creates ~1.4M nodes.
+
+References: 
+    -- https://github.com/MISP/misp-warninglists
+*/
+
+// Create and tag CIDR indicators. 
+function createNodes(content, name){
+    for $ioc in $content.list{
+        if ($ioc ~= '/'){
+            for $form in ('inet:cidr4', 'inet:cidr6'){
+                [ *$form?=$ioc +#warninglist.$name ]
+            }
+        }
+        else {
+            for $form in ('inet:ipv4', 'inet:ipv6'){
+                [ *$form?=$ioc +#warninglist.$name ]
```

Comment:
may want to adjust this to attempt to create the inet:ipv4, and if successful skip creating the inet:ipv6. currently it will also create inet:ipv6 nodes for ipv4 addresses which may not be intended

Reply:
@Cisphyx - can you recommend the "best" way to implement that ipv4 creation check?